### PR TITLE
sameboy: 0.13.6 -> 0.14.1

### PR DESCRIPTION
**Pull request by meithecatte**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114583` at 1aa90bbb run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/353e7fe9e652d0e289714f0b673f080a">sameboy</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md">patch-phase</a>

`patchPhase` should not be overridden, use `postPatch` instead.

Near pkgs/misc/emulators/sameboy/default.nix:22:3:
```
   |
22 |   patchPhase = ''
   |   ^
```
</li>
  </ul>
</details>

**Comment by NieDzejkob** [comment]
Thanks, fixed the suggestion.

File: pkgs/misc/emulators/sameboy/default.nix [hunk 1]
```diff
@@ -2,34 +2,30 @@
...
   };
 
   enableParallelBuilding = true;
   # glib and wrapGAppsHook are needed to make the Open ROM menu work.
   nativeBuildInputs = [ rgbds glib wrapGAppsHook ];
   buildInputs = [ SDL2 ];
 
-  makeFlags = "CONF=release DATA_DIR=$(out)/share/sameboy/";
+  makeFlags = [ "CONF=release" "CAN_INSTALL=true" "PREFIX=$(out)"
+                "DATA_DIR=$(out)/share/sameboy/" ];
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/sameboy/default.nix:20]:
```suggestion
  makeFlags = [
    "CAN_INSTALL=true"
    "CONF=release"
    "DATA_DIR=$(out)/share/sameboy/"
    "PREFIX=$(out)"
  ];
```

File: pkgs/misc/emulators/sameboy/default.nix [hunk 2]
```diff
@@ -2,34 +2,30 @@
...
 
-  patchPhase = ''
+  postPatch = ''
     sed 's/-Werror //g' -i Makefile
     sed 's@"libgtk-3.so"@"${gtk3}/lib/libgtk-3.so"@g' -i OpenDialog/gtk.c
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/sameboy/default.nix:24]: This should use substituteInPlace.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/misc/emulators/sameboy/default.nix (hunk 2 above)
Comment (NieDzejkob) [reply pkgs/misc/emulators/sameboy/default.nix]: Pushed a fix.

**Comment by NieDzejkob** [comment]
What's the status on this PR? Is it blocked on me or review?

**Comment by SuperSandro2000** [comment]
> What's the status on this PR? Is it blocked on me or review?

I just have 900 GitHub notifications and didn't look at them the last two days.

**Comment by NieDzejkob** [comment]
On Mon Mar 1, 2021 at 5:48 PM CET, Sandro wrote:
> > What's the status on this PR? Is it blocked on me or review?
>
> I just have 900 GitHub notifications and didn't look at them the last two days. 

Totally understandable!Have a nice day.

**Pull request merged**